src/main/java/com/dodatabase/movie_backend/domain/Movie/NationType.java

```diff
@@ -0,0 +1,19 @@
+package com.dodatabase.movie_backend.domain.Movie;
+
+import lombok.Getter;
+
+@Getter
+public enum NationType {
+    대한민국("대한민국"),
+    미국("미국"),
+    일본("일본"),
+    홍콩("홍콩"),
+    영국("영국"),
+    프랑스("프랑스");
```

Comment:
good changing.

Reply:
?? : no, i don't think so

Reply:
He modified many countries to just 6 countries which have big influence at movie field.
If you want to use another country, just leave blank.